app/workers/moderator/sink_articles_worker.rb

```diff
@@ -0,0 +1,20 @@
+module Moderator
+  class SinkArticlesWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
```

Comment:
Not sure if this should be high priority.

Reply:
I would say `medium_priority` I like to think of high_priority items as things users do in the UI and expect feedback on quickly like updating an article, they expect that cache to bust very quickly. Since its our internal moderators doing this in the background I dont think its as urgent.